features/admin.applications.v1/components/help-panel/mtls-oidc-configurations.tsx

```diff
@@ -0,0 +1,165 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { AsgardeoSPAClient, OIDCEndpoints } from "@asgardeo/auth-react";
+import { AlertLevels, TestableComponentInterface } from "@wso2is/core/models";
```

Comment:
TestableComponentInterface is deprecated.

```suggestion
import { AlertLevels, IdentifiableComponentInterface } from "@wso2is/core/models";
```

Reply:
fixed with [3a0b2fe](https://github.com/wso2/identity-apps/pull/6097/commits/3a0b2fee82c33f4d8e81dec05d773c557332182c)